front/app/modules/commercial/custom_maps/utils/tileProviderDefaultOptions.ts

```diff
@@ -14,8 +14,8 @@ export const BASEMAP_AT_DEFAULT_OPTIONS = {
   tileSize: 256,
   crossOrigin: true,
   minZoom: 1,
-  maxZoom: 18,
+  maxZoom: 20,
   subdomains: ['maps', 'maps1', 'maps2', 'maps3'],
   attribution:
-    'Karte: <a href="http://basemap.at" target="_blank">basemap.at</a>, <a href="http://creativecommons.org/licenses/by/3.0/at/deed.de" target="_blank">CC-BY 3.0</a>',
+    'Karte: <a href="http://basemap.at" target="_blank">basemap.at</a>, <a href="https://creativecommons.org/licenses/by/4.0/deed.de" target="_blank">CC-BY 4.0</a>',
```

Comment:
Would it make sense to add this to Crowdin somehow so that it doesn't need a code change when it has to be changed?

Reply:
Maybe, but we might also want to make the attribution editable in the back office as part of a future [map configurability](https://citizenlab.atlassian.net/browse/EN-1730) epic. This hardcoded stuff is kind of a temporary solution